### PR TITLE
Add print notes script and make level argument optional

### PR DESCRIPTION
**Pull request by kemartin2015**
Hi @helrond, when you get a chance, can you look at the change to the logic I made in line 31 of the edit notes script to make the level argument optional? I made the same change to the print_notes.py script. Thanks!

### CODE REVIEW

**Review by helrond — COMMENTED** [review]
Thanks @kemartin2015 - I think what you have here should work. Looking at the code though, I'm wondering if I made the right suggestion in terms of making the `level` argument optional, because it means if it's not present then the argument value will be `None`, which is semantically kind of the opposite of what we want. 

Off the top of my head, I think there are two potential tweaks:
- keep the `level` arg optional, but add a default (something like `all`) which can then be checked in that `if` block.
- make the `level` arg required, but add another option (again, something like `all`) to the `LEVEL` list, which would again be checked in that same place. 

The net effect of either of those approaches is that the logic in the `if` statement is a bit more apparent to someone reading the code for the first time. Does that makes sense?

**Comment by kemartin2015** [comment]
This makes sense, @helrond. Your suggestions would make it much easier for someone reading the code for the first time. I'll play around with both options. Thank you!

**Comment by kemartin2015** [comment]
@helrond I think for the sake of future projects, I like having to specify the level when running the script. I made the level argument required and added `all` as one of the choices. Do my changes look okay?

File: archivessnake/edit_notes.py [hunk 1]
```diff
@@ -20,15 +20,15 @@
...
+LEVEL = ["collection", "file", "series", "item", "all"]
 CONFIDENCE_RATIO = 97
 
 def process_tree(args, resource):
     """Iterates through a given collection, file, or series provided by user input. Finds note content that matches user input and then deletes or modifies relevant notes according to user preference.  """
     for record in resource.tree.walk:
         updated = False
         aojson = record.json()
-        if record.level == args.level:
+        if record.level == args.level or args.level == "all":
```
Comment (helrond) [inline archivessnake/edit_notes.py]: I believe you can do
```suggestion
        if record.level in [args.level, "all"]:
```

**Review by helrond — COMMENTED** [review]
This makes sense to me!

File: archivessnake/edit_notes.py (hunk 1 above)
Comment (kemartin2015) [reply archivessnake/edit_notes.py]: I changed it to `if args.level in [record.level, "all"]:` and that worked!

**Review by helrond — APPROVED** [review]

**Pull request merged**